worker/uniter/pebblepoller_test.go

```diff
@@ -141,3 +144,42 @@ func (c *fakePebbleClient) CloseIdleConnections() {
 	defer c.mut.Unlock()
 	c.closed = true
 }
+
+// AddNotice adds a notice for WaitNotices to receive. To have WaitNotices
+// return an error, use notice.Type "error" with the message in notice.Key.
+func (c *fakePebbleClient) AddNotice(checkC *gc.C, notice *pebbleclient.Notice) {
+	select {
+	case c.noticeAdded <- notice:
+	case <-time.After(time.Second):
```

Comment:
We typically use testing.ShortWait

Reply:
I've updated to `testing.LongWait` as that seems appropriate here from its comment: it's something which should happen (and will normally take a small amount of time, but on hiccup could take longer). Whereas `ShortWait` is for things which shouldn't happen and we just want to confirm that 99% of the time.